rviz_plugins/eStop_plugin/src/bare_panel.cpp

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <string>
+
+#include <QVBoxLayout>
+#include <QHBoxLayout>
+#include <QLineEdit>
+
+#include "bare_panel.h"
+
+namespace bare
+{
+
+void BarePanel::subCallback(const std_msgs::Bool& msg) {
+	if (msg.data) {
+		output_topic_editor_->setStyleSheet("QLabel {color : green;}");
+		output_topic_editor_->setText("enabled");
+	}
+	else {	
+		output_topic_editor_->setStyleSheet("QLabel {color : red;}");
+		output_topic_editor_->setText("disabled");
+	}
+}
+
+BarePanel::BarePanel( QWidget* parent )
+  : rviz::Panel( parent )
+{
+	QHBoxLayout* topic_layout = new QHBoxLayout;
+	topic_layout->addWidget( new QLabel( "eStop Status:" ) );
+	output_topic_editor_ = new QLabel("TEST");
```

Comment:
The QLabel constructor doesn't have to have any arguments.
The label can remain empty until the first callback is called or it could start as "enabled."